src/App.tsx

```diff
@@ -40,7 +40,7 @@ function App() {
           <Layout>
             <Routes>
               <Route path={"/login/success"} element={<LoginRedirect />} />
-              <Route path={"/"} element={<PublicHome />} />
+              <Route path={"/"} element={<LoginRedirect />} />
```

Comment:
Public home should be served on "/" right?

Reply:
Is it? I thought that when we first login or open the application we should be redirected to the profile page?

Reply:
Yeah but for the people who aren't logged in still public home should be visible